Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.java

```diff
@@ -0,0 +1,90 @@
+/*
+    LONGEST PALINDROMIC SUBSEQUENCE
+
+    The task is to find the length of the longest possible subsequence of a given word which is      a palindrome.
+    A palindrome is a string that is same when read from any side.
+
+*/
+public class Solution {
+
+    /*
+    We use Dynamic Programming approach to solve this problem.
+    A 2D matrix M of size n*n is created where n is the length of the given string.
+    Here the coulmn j denotes the char S[j] of the string while row i denotes char R[i], where
+    R is the reverse of the string S.
+    Thus, M[i][j] denotes the max length of longest common subsequence between the string 
+    S[0...j] and R[0..i]. Hence M[n][n] will compute the longest common subsequence of S[0..n]       and R[0..n], which is the required answer.
```

Comment:
Remove the extra spaces between **S[0..n]** and **and**.

Divide the comment into more lines. Try to maintain a limit of 80 characters per line.

Do the same in C++ and Python code.